### DISCUSSION TITLE
Wildcards In Filtering?

### DISCUSSION THREAD

**Discussion by Majoraslayer** [containrrr/watchtower#1986] [Q&A]
I have a bunch of containers in my homelab, and Watchtower does a great job of keeping them updated.  However, a couple of the applications I use depend on stacks of containers they control (specifically Nextcloud and Kasm), and I'd like to exclude them from update checks.  Nextcloud is pretty easy, as I can just list the containers for it in the WATCHTOWER_DISABLE_CONTAINERS variable.  However, Kasm often creates new containers for its own use with somewhat unique names that follow a pattern.  Particularly, it creates kasm_volume_helper_hOIHDO with a random string at the end, and Watchtower always emails me errors when it tries to update these.  Is there any way to set up a filter with a wildcard so it would know to skip any containers with a name following a pattern, such as kasm_*?

**Comment by TristisOris** [comment]
Also need wildcard for excludes.

**Comment by sandipb** [comment]
Same here. I need filtering for the k8s nodes created by Kind. I cannot get kind to label them. And their docker images are generally sha256 hash based. I keep getting a lot of watchtower spam notifications for these containers as a result.
